### ISSUE TITLE
Integrate Traceur (source map support)

### ISSUE THREAD

**Issue by guybedford** [ModuleLoader/es6-module-loader#31]
I've been considering how to go about polyfilling ES6 functionality dynamically (classes, etc) with this module, and there is a development workflow that could make sense.

Particularly, I was considering how one could load https://github.com/addyosmani/traceur-todomvc without a build step when in development.

The basic idea would be to compile the ES6 with Traceur before execution. For an example of this see https://github.com/guybedford/es6.

Basically it would be added as a "translate" step in the loader:

``` javascript
  System.translate = function(source) {
    // copy code from https://github.com/guybedford/es6/blob/master/es6.js#L4
  }
```

The integration is actually quite simple as all the code is readily available. The question is whether we want to encourage this or not?
